src/pages/patientView/clinicalInformation/lib/clinicalAttributesUtil.js

```diff
@@ -132,6 +132,18 @@ function derive(clinicalData) {
                     caseTypeNormalized = 'cfDNA';
                 } else if (caseTypeLower.indexOf('prim') >= 0) {
                     caseTypeNormalized = 'Primary';
+                } else if (caseTypeLower.indexOf('ctdna') >= 0) {
+                    caseTypeNormalized = 'ctDNA';
+                } else if (caseTypeLower.indexOf('plasma') >= 0) {
+                    caseTypeNormalized = 'Plasma';
+                } else if (caseTypeLower.indexOf('urine') >= 0) {
+                    caseTypeNormalized = 'Urine';
+                } else if (caseTypeLower.indexOf('exosome') >= 0) {
+                    caseTypeNormalized = 'Exosome';
+                } else if (caseTypeLower.indexOf('total') >= 0) {
```

Comment:
Maybe this is a data-related question, not sure if you know the answer, `total` refers to `total RNA` which seems strange, is this defined somewhere in our documentation?

Reply:
curious about this too

Reply:
doesn't really feel like a sample type? Feels like RNA should be deduced from the profiling rather than sample name